packages/es-components/src/components/controls/textbox/Textbox.js

```diff
@@ -182,9 +192,10 @@ const Textbox = props => {
       color={theme.validationTextColor[validationState]}
       inline={inline}
     >
-      <LabelText className="es-textbox__label" inline={inline}>
+      <StyledLabelText className="es-textbox__label" inline={inline}>
         {labelText}
-      </LabelText>
+        {optional && <OptionalTag> Optional</OptionalTag>}
```

Comment:
It would be nice to allow an element that could be styled differently and contain different text, though right now there's only the one design. I'll change it so it's more future-proof.